osmtogtfs/gtfs/gtfs_dummy.py

```diff
@@ -46,12 +60,13 @@ def _create_dummy_calendar():
              'friday': 1, 'saturday': 0, 'sunday': 0, 'start_date': 20170101, 'end_date': 20190101}]
 
 
-def _create_dummy_trips(routes, stops, route_stops, calendar):
+def _create_dummy_trips(routes, stops, stops_per_route, calendar):
     trips = []
```

Comment:
@mehdisadeghi from my experience optimizing Clojure code I can say that it is probably more efficient to let Python built the trips datastructure instead of manually append items 1 by 1. So I guess something like `trips = [t for t in _create_dummy_trips(...)]` would be faster since Python can optimize things internally